tests/integration/components/models-table-test.js

```diff
@@ -2995,3 +2997,72 @@ test('#grouped-rows #column custom group-cell component actions', function (asse
   assert.ok(ModelsTableBs.getRowsFromGroupColumn(0).every(r => r.expanded), 'All rows for rows group become expanded');
   assert.equal(groupingRowsByColumn(0).expandedCountText, firstGroupRowsCount);
 });
+
+test('in-line edit: row is editable, column displays default edit component ', function(assert) {
+
+  assert.expect(13);
+
+  this.register('component:stub-comp-edit',
+    Component.extend({
+      classNames: ['cellInput'],
+      layout: hbs`{{get record propertyName}}`
+    })
+  );
+
+  const columns = generateColumns(['index', 'firstName', 'lastName']);
+  columns[0].editable = false; // Index is not editable
+  columns[1].componentForEdit = "stub-comp-edit"; // Index is not editable
+
+  this.setProperties({
+    data: generateContent(5, 1),
+    columns
+  });
+
+  this.render(hbs`
+    {{#models-table data=data columns=columns as |c|}}
+      {{#c.table as |table|}}
+        {{#table.body as |body|}}
+          {{#each body.visibleContent as |record index|}}
+            {{#body.row record=record index=index as |row|}}
+            {{log row}}
```

Comment:
Remove pls

Reply:
Yea I see that. I forgot to take that out. Odd though that it would fail a test, and not a test its in.

Do you want me to remove it and update the pull request?

Reply:
Yes, pls remove it.